src/components/Upload/UploadButton/UploadButton.jsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import history from '../../../services/history';
+import { withRouter } from "react-router";
 
 export class UploadButton extends React.Component {
     goToUpload() {
-        history.push('/upload');
+        
```

Comment:
Added an extra line, also, is this file actually used?

Reply:
The history file is still used in the redux actions